README.md

```diff
@@ -92,14 +93,32 @@ pre-commit install
 
 ```
 
+For the annotation, use the following lines to load and activate that correct environment:
```

Comment:
I think we need to explicitly explain to the users what they will and won't be able to do when using either environment. I'm happy to change the README.md afterwards, there are other documentation issues that need to be added 😃